src/test/java/com/google/opensesame/mockObjects/MockGHObjectsTest.java

```diff
@@ -0,0 +1,32 @@
+package com.google.opensesame.mockObjects;
```

Comment:
These tests might not be necessary because they are testing getter code. Is there a particular reason why you think these are necessary?

Reply:
I'm not super familiar with Mockito and I wanted to be sure that my assumptions for the interface would work. I caught an error with them too

Reply:
I could definitely remove them if you think that makes sense.

Reply:
I think they can probably be removed, but feel free to wait and get a second opinion from Bill or Joe before you delete it.

Reply:
I think it's fine to keep for now. 1 other thing unit tests do is document the usage of classes.